scripts/docx_adapter.py

```
@@ -0,0 +1,109 @@
+    def extract_segments(self, file_path: str) -> List[Segment]:
+        """Extract text segments from DOCX file."""
+        segments = []
+        docx_path = Path(file_path)
+
+        with zipfile.ZipFile(docx_path, 'r') as zf:
+            # Read document.xml
+            with zf.open('word/document.xml') as f:
+                xml_content = f.read()
+
+            if len(xml_content) > self.MAX_XML_SIZE:
+                raise ValueError("XML content too large")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)